website/.env.example

```diff
@@ -0,0 +1 @@
+BEEHIIV_API_KEY=L4cvXwCqHADQ9DI4XQCHh7JSZIOE79qU7kBXysTOWmAnBWIXV3wzRfyTA12wt6Nn
```

Comment:
remove?

Reply:
Yes, I've removed this, also related to above context, it's now just a placeholder